src/runtime/handle.ts

```diff
@@ -138,6 +138,17 @@ export abstract class Handle {
   protected generateKey(): string {
     return this.idGenerator.newChildId(Id.fromString(this._id), 'key').toString();
   }
+
+  disable(particle?: Particle, storageProxy?: StorageProxy): void {
+    if (storageProxy != undefined) {
```

Comment:
I think we want `!==` which checks both value and type. More at:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Comparison_Operators#Nonidentity
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Expressions_and_Operators

but also I thought of it because I know much other code in this area is like this.